### DISCUSSION TITLE
ESlint not showing code error/warning on my vscode

### DISCUSSION THREAD

**Discussion by borisBelloc** [eslint/eslint#14341] [Q&A]
Hello, :wave:

I develop with Angular (typescript) and I'm using ESLint on VSCode.

Since 1-2months, ESLint stopped showing warnings on my code when I do an error (missing semi-colon, wrong indentation, etc...).

I have to confess, I don't really know if ESLint stopped working or if my linter stopped working when I switched from TSLint to ESLint...

I don't know where to find help, so I am hoping someone here can help me to configure ESLINT.

I installed Vscode's ESLint extension.
My ESLint is activated... I don't find how to fix it.

Screenshots : 

**VSCode extensions + setting :** 
![image](https://user-images.githubusercontent.com/37147299/115140576-3869c280-a038-11eb-872d-27bf186c8248.png)

ESLint status + missing warnings on TypeScript file 
In this screen, I'm expecting ESLint to show me a warning (yellow underline) because of the 2 missings `;` (line 20 and 26) and a warning for the bad indentation line 24: 
![image](https://user-images.githubusercontent.com/37147299/115140668-bb8b1880-a038-11eb-8ca5-f834690eca10.png)

Any idea what can I do?
Thank you

**Comment by nzakas** [comment]
We don’t maintain the VS Code extension. You may want to ask those folks for help: https://github.com/microsoft/vscode-eslint

**Reply by borisBelloc** [reply]
Ok, I didn't know the problem was from the extension and not from ESLint.
I did read that ESLint need the extension to work on VSCode so I thought it was the same thing.
Thank you.
